trading_calendars/exchange_calendar_xnys.py

```diff
@@ -65,7 +65,7 @@ class NYSEExchangeCalendar(TradingCalendar):
     - Thanksgiving (fourth Thursday in November)
     - Christmas (observed on nearest weekday to December 25)
 
-    NOTE: The NYSE does not observe the following US Federal Holidays:
+    NOTE: The XNYS does not observe the following US Federal Holidays:
```

Comment:
I'm not sure we need to update the lines in this docstring. I think most people will read this easier as NYSE than as XNYS.